includes/class-meta.php

```diff
@@ -33,16 +33,15 @@ public static function init() {
 	 */
 	public static function register_option() {
 		$type   = static::get_schema()['type'] ?? 'string';
+		$cap    = static::get_capability();
```

Comment:
Isn't this line superfluous? `$cap` is never used here.

Reply:
yes! left over from a previous interaction. thanks for spotting